tests/Php81/Php81Test.php

```diff
@@ -34,6 +34,7 @@ public function testArrayIsList()
 
     /**
      * @requires extension mysqli
+     * @requires PHP < 8.4
```

Comment:
This test should be executed on PHP 8.4 as well. We should rather allow the test to trigger deprecations.

Reply:
Right